src/js/components/Heading/Heading.js

```diff
@@ -34,6 +46,13 @@ const Heading = forwardRef(
       return () => window.removeEventListener('resize', updateOverflowWrap);
     }, [headingRef, overflowWrapProp]);
 
+    let content = children;
+    if (skeleton) {
+      content = (
+        <HeadingSkeleton level={level} />
```

Comment:
Should we be passing `size` here as well? It looks like size is used in the HeadingSkeleton file

Reply:
Yup. Adding actually `fill` and `...rest`. Just don't need the weight or children, etc.